apps/ui/src/icons/HistoryIcons.tsx

```diff
@@ -0,0 +1,57 @@
+import { IconProps } from "./types";
+
+export const HistoryActive = (props: IconProps) => {
+  return (
+    <svg
```

Comment:
Not for this PR - but we need to move away from INLINE svgs and move them into the public dir as real images.